tests/test_tolino_note.py

```diff
@@ -169,3 +169,20 @@ def test_lang_de_noted_note_3(self) -> None:  # noqa: D102
         assert note.user_notes
         assert note.user_notes.startswith('Test Note')
         assert note.user_notes.endswith('quotes" at beginning.')
+
+    def test_issue_dkuester_github_3(self) -> None:  # noqa: D102
+        note = TolinoNote.from_unparsed_content(
+            """Miss Merkel (Safier, David)
+Markierung"In diesem Moment realisierte Angela, dass sie zwar während ihrer ganzen
```

Comment:
For some reason this note does not contain a page number and misses the leading whitespace. *sigh*